promgen/templates/promgen/exporter_form.html

```diff
@@ -11,7 +11,7 @@ <h1>Project: {{ project.name }}</h1>
 
 <div class="row">
 
-<form id="exporter_form" action="{% url 'project-exporter' project.id %}" method="post">{% csrf_token %}
+<form id="exporter_form" action="{{ url }}" method="post">{% csrf_token %}
```

Comment:
I think you can just remove `action="{{ url }}"` and let it default to POSTing to the same page. Then you don't need to do extra work in `get_context_data`

Reply:
`url` is also used in `ExporterRegister`, so I cannot remove `action="{{ url }}"`
https://github.com/line/promgen/pull/262/files#diff-4148d3344e097bea3c3c8f89325b0f57R629

Reply:
Yes, I think you can remove it from both :)

Reply:
I'm sorry. I understood.
fixed https://github.com/line/promgen/pull/262/commits/d39efd4940a5f02f32f9c127a21e44ee3e6f0a76